### DISCUSSION TITLE
STM32 or dwc2 Host MSC

### DISCUSSION THREAD

**Discussion by mcuenthusiast** [hathach/tinyusb#2183] [Q&A]
Hi All,

Does any one have implemented stm32 host MSC? I am using stm32U I have below reference code for host but not working with MSC it works with HID.

https://gist.github.com/Ryzee119/5a1b74e98449fdf5ed24b3f361ff4e2c.

Any help will be highly appreciated.

Thanks in advance

**Comment by mcuenthusiast** [comment]
@hathach any plan for adding the dwc2 host support since many devices are using the dwc2 driver.

**Comment by hathach** [comment]
I don't plan anymore, I will only implement feature when needed with my fulltime work or when I think I could manage to implement a feature within my spare time.
